drake/systems/plants/RigidBodyTreeURDF.cpp

```diff
@@ -563,8 +563,21 @@ void parseTransmission(RigidBodyTree * model, XMLElement* node) {
   double gain = 1.0;
   if (reduction_node) parseScalarValue(reduction_node, gain);
 
-  RigidBodyActuator a(actuator_name, model->bodies[body_index], gain);
-  model->actuators.push_back(a);
+  XMLElement *limit_node = joint_node->FirstChildElement("limit");
+  double effort_min = -numeric_limits<double>::infinity();
+  double effort_max = numeric_limits<double>::infinity();
+  if (limit_node) {
+    bool effort_min_or_max_specified = false;
+    effort_min_or_max_specified = effort_min_or_max_specified || parseScalarAttribute(limit_node, "effort_min", effort_min);
+    effort_min_or_max_specified = effort_min_or_max_specified || parseScalarAttribute(limit_node, "effort_max", effort_max);
+
+    if (!effort_min_or_max_specified) {
+      parseScalarAttribute(limit_node, "effort_min", effort_max);
```

Comment:
Fixed in https://github.com/rdeits/drake/pull/28. Sorry about that, that was very wrong; good catch.